packages/node/test/integration/cross-node-multisig-creation.spec.ts

```diff
@@ -39,13 +40,15 @@ describe("Node can create multisig, other owners get notified", () => {
       messagingService,
       storeService,
       EMPTY_NETWORK,
+      ethers.getDefaultProvider(),
```

Comment:
This uses Infura I think, probably not best for a test

Reply:
WIP - just wanted to get it working at the moment - should pass in a mock provider